actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.sunbird.feed;
+
+import java.util.*;
+import java.util.stream.IntStream;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.bean.ShadowUser;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.feed.impl.FeedFactory;
+import org.sunbird.models.user.Feed;
+
+/** this class will be used as a Util for inserting Feed in table */
+public class FeedUtil {
+  private static IFeedService feedService = FeedFactory.getInstance();
+  private static final String ORG_MIGRATION_ACTION = "OrgMigrationAction";
+  public static final String UNREAD = "unread";
+  public static final String READ = "read";
+
+  public static void saveFeed(ShadowUser shadowUser, List<String> userIds) {
+    Map<String, Object> reqMap = new HashMap<>();
+    reqMap.put(JsonKey.USER_ID, userIds.get(0));
+    reqMap.put(JsonKey.CATEGORY, ORG_MIGRATION_ACTION);
+    List<Feed> feedList = feedService.getRecordsByProperties(reqMap);
+    int index = getIndexOfMatchingFeed(feedList);
+    if (index == -1) {
+      feedService.insert(getFeedObj(shadowUser, userIds));
+    } else {
+      Map<String, Object> data = feedList.get(index).getData();
+      List<String> channelList = (List<String>) data.get(JsonKey.PROSPECT_CHANNELS);
+      channelList.add(shadowUser.getChannel());
+      feedService.update(feedList.get(index));
+    }
+  }
+
+  private static Feed getFeedObj(ShadowUser shadowUser, List<String> userIds) {
```

Comment:
1 .getFeedObj giving sense , we are fetching feed data  from some storage, but here we are just creating feedObject , put some related name.
2. no need to take list of userIds as we are consuming only one id.